src/stateToHTML.js

```diff
@@ -263,18 +263,27 @@ class MarkupGenerator {
     return this.blocks[this.currentBlock + 1];
   }
 
-  writeStartTag(blockType, customRendererOptions) {
+  writeStartTag(blockType, customRendererOptions, blockData = {}) {
     let tags = getTags(blockType);
 
     let attrString = '';
+    let attributes = {};
     if (customRendererOptions) {
-      let attributes = customRendererOptions && customRendererOptions.attributes;
-      attrString = stringifyAttrs(attributes);
+      attributes = customRendererOptions.attributes || {};
     }
 
+    const alignment = blockData.get('alignment');
+    if (alignment) {
+      attributes.class = attributes.class || '';
+      attributes.class += ` block--align-${alignment}`.trim();
```

Comment:
I do, but your code is also broken because I would be left with the untrimmed version

    > a = "  "
    '  '
    > (a += 'xd').trim()
    'xd'
    > a
    '  xd'

I'll fix it, good catch